inbm/Changelog.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ## NEXT - ?
 
 ### Fixed
+ - RTC 509640      - Shorten the SWBOM publish time and added logic to cancell / update timer
```

Comment:
'cancel' only has one 'l'.  This has two.

Reply:
comment has both present (shorten) and past (added) tense verbs.  Change to one or the other.  Maybe
'Shortened the SWBOM publish timer and added logic to cancel and update the timer'.

Reply:
updated as per comment